static/lang/en.json

```diff
@@ -734,7 +734,6 @@
         "ArmorPropertyRuneEthereal": "Ethereal",
         "ArmorPropertyRuneFireResistant": "Fire-Resistant",
         "ArmorPropertyRuneFortification": "Fortification",
-        "ArmorPropertyRuneGlamered": "Glamered",
```

Comment:
Is there a reason that Glamered rune is gone?

Reply:
I see, it's from an unrelated PR. You should probably remove this change from here and keep it in that PR.

Reply:
Removed the change.